app/helpers/application_formatters_helper.rb

```diff
@@ -114,6 +114,19 @@ def format_duration(total_seconds)
     format('%02dH%02dM%02dS', hours, minutes, seconds)
   end
 
+  # @return the duration in the format for displaying, eg 02:45:12
+  def format_duration_for_display(total_seconds)
+    seconds = total_seconds % 60
```

Comment:
Since it is extremely unlikely that the duration will be more than 24h, would it be cleaner to do ([ref](https://stackoverflow.com/questions/4175733/convert-duration-to-hoursminutesseconds-or-similar-in-rails-3-or-ruby)):

```ruby
format_string = total_seconds < 3600 ? '%M:%S' : '%H:%M:%S'
Time.at(total_seconds).utc.strftime(format_string)
```

Nitpick: can add the return type for the comments?: `# @return [String] ...`. (We should change the previous method too.)

Reply:
Alright, will update.